requirements-test.txt

```diff
@@ -0,0 +1 @@
+openwisp-utils[qa]>=0.2.1
```

Comment:
this is not needed

Reply:
hmm installation with travis without requirements-test.txt files